lib/_tls_wrap.js

```diff
@@ -19,7 +19,7 @@ const defaultSessionIdContext = getDefaultSessionIdContext();
 function getDefaultSessionIdContext() {
   var defaultText = process.argv.join(' ');
   /* SSL_MAX_SID_CTX_LENGTH is 128 bits */
-  if (process.config.variables.openssl_fips) {
+  if (process.config && process.config.variables && process.config.variables.openssl_fips) {
```

Comment:
;-) yeah, I was being lazy. Sorry about that. As I said, the linting nit can be addressed when the PR is landed.